deepchem/models/torch_models/grover_layers.py

```diff
@@ -262,6 +262,8 @@ def forward(self, embeddings: Dict, atom_scope: List, bond_scope: List):
         """
         preds_atom_from_atom, preds_atom_from_bond, preds_bond_from_atom, preds_bond_from_bond = None, None, None, None
 
+        # TODO remove if condition
+        # see https://github.com/deepchem/deepchem/pull/3300
```

Comment:
Should this be removed since #3300 has been merged in?

Reply:
Its a leftover cleanup from the earlier PR which I would like to do it in a different pull request.